src/structure/model.jl

```diff
@@ -0,0 +1,1089 @@
+organisestructure(chains::Vector{Chain}; structure_name::ASCIIString="", model_number::Int=1) = organise(organisemodel(chains; model_number=model_number); structure_name=structure_name)
+organisestructure{T <: AbstractResidue}(residues::Vector{T}; structure_name::ASCIIString="", model_number::Int=1) = organise(organisemodel(residues; model_number=model_number); structure_name=structure_name)
+organisestructure{T <: AbstractAtom}(atoms::Vector{T}; structure_name::ASCIIString="", model_number::Int=1) = organise(organisemodel(atoms; model_number=model_number); structure_name=structure_name)
+organisestructure(model::Model; structure_name::ASCIIString="") = organisestructure([model]; structure_name=structure_name)
+organisestructure(chain::Chain; structure_name::ASCIIString="", model_number::Int=1) = organisestructure([chain]; structure_name=structure_name, model_number=model_number)
+organisestructure(res::AbstractResidue; structure_name::ASCIIString="", model_number::Int=1) = organisestructure(AbstractResidue[res]; structure_name=structure_name, model_number=model_number)
+organisestructure(atom::AbstractAtom; structure_name::ASCIIString="", model_number::Int=1) = organisestructure(AbstractAtom[atom]; structure_name=structure_name, model_number=model_number)
+
+
+"""
+Determines if an `AbstractAtom` represents a non-hetero atom, i.e. came from an
+ATOM record.
+"""
+stdatomselector(atom::AbstractAtom) = !ishetatom(atom)
+
+"""
+Determines if an `AbstractAtom` is a hetero atom, i.e. came from a HETATM
+record.
+"""
+hetatomselector(atom::AbstractAtom) = ishetatom(atom)
+
+"""
+Determines if an `AbstractAtom` has its atom name in the given `Set` or
+`Vector`.
+"""
+atomnameselector(atom::AbstractAtom, atom_names::Set{ASCIIString}) = atomname(atom) in atom_names
+# Set is faster but Vector method is retained for ease of use
+atomnameselector(atom::AbstractAtom, atom_names::Vector{ASCIIString}) = atomname(atom) in atom_names
+
+"`Set` of C-alpha atom names."
+const calpha_atom_names = Set(["CA"])
+
+"""
+Determines if an `AbstractAtom` is not a hetero-atom and corresponds to a
+C-alpha atom.
+"""
+calphaselector(atom::AbstractAtom) = stdatomselector(atom) && atomnameselector(atom, calpha_atom_names)
+
+"`Set` of protein backbone atom names."
+const backbone_atom_names = Set(["CA", "N", "C"])
+
+"""
+Determines if an `AbstractAtom` is not a hetero-atom and corresponds to a
+protein backbone atom.
+"""
+backboneselector(atom::AbstractAtom) = stdatomselector(atom) && atomnameselector(atom, backbone_atom_names)
+
+"`Set` of protein standard heavy atom names."
+const heavy_atom_names = Set(["C", "CA", "CB", "CD", "CD1", "CD2",
+                    "CE", "CE1", "CE2", "CE3", "CG", "CG1", "CG2", "CH2", "CZ",
+                    "CZ2", "CZ3", "N", "ND1", "ND2", "NE", "NE1", "NE2", "NH1",
+                    "NH2", "NZ", "O", "OD1", "OD2", "OE1", "OE2", "OG", "OG1",
+                    "OH", "SG"])
```

Comment:
I want to know where you got this heavy atom list from. Inserting a comment about source like https://github.com/BioJulia/Bio.jl/blob/master/src/seq/geneticcode.jl#L35-L36.